### PR TITLE
creating a navigation bar

### PR DESCRIPTION
**Pull request by EdwardVillasenor1992**
**1. Brief Summary of what this PR accomplishes (140 characters or less. If you find trouble describing what you are doing in this length, consider breaking the PR into multiple ones.)**

created a simple navigation bar for the web page as a starting point. Also updated some of the CSS for the nav bar.

**2. Link to Trello Ticket**

https://trello.com/c/zPMpJObT/69-create-navigation-for-site

**3. More detailed description and other questions to address in code review**
sample

**4. Remember to tag reviewers!**
@erikeldridge @JessDF @RaquelFigueroa

### CODE REVIEW

**Comment by RaquelFigueroa** [comment]
I really like the design!
From the standpoint of consistency, not all of the options on the navbar are capitalized. I suggest staying consistent and all options all lower-case or capitalize first letter.
Other than that, looks good to me!

**Comment by JessDF** [comment]
It looks really good, I agree with Raquel's statement. 

Quick question - do the options go anywhere?

**Comment by erikeldridge** [comment]
+1 to Bennett's recommendation.

We're a small, variable team, so we have limited resources for maintenance. Biasing to established libs/patterns reduces our maint cost.

We've done the work to integrate bootstrap-react, which provides responsive nav, and is tested by a large community. By defining our own nav and responsiveness logic we're not taking advantage of this and reducing consistency.

Could we use `Col` elements to layout the side bar and content areas?
https://react-bootstrap.github.io/layout/grid/ 

Could we use a stacked `Nav` as structure for the left bar?
https://react-bootstrap.github.io/components/navs/#navs-stacked

Given this, the custom css we have to maintain could be reduced to just overriding `NavItem` styling to fit the look you have.

File: webapp/components/App.jsx [hunk 1]
```diff
@@ -45,25 +45,34 @@ class App extends Component {
...
                     <Row>
                         <div className="contribution-list">
-                            <p>
-                                {'Below are some candidate contributions loaded '}
-                                {'form the API server. Now let\'s visualize them!'}
-                            </p>
+                        <p>
+                            Below are some candidate contributions loaded from
+                            the API server. Now let's visualize them!
+                        </p>
```
Comment (JessDF) [inline webapp/components/App.jsx]: Just to keep in mind - you should run the linter to make sure spacing and everything is up to standards. The command is: `yarn run lint`

**Pull request closed without merging**